apps/vr-tests/src/stories/Badge.stories.tsx

```diff
@@ -1,11 +1,11 @@
 import { storiesOf } from '@storybook/react';
 import * as React from 'react';
-import { Badge, BadgeCommons } from '@fluentui/react-badge';
+import { Badge, BadgeProps } from '@fluentui/react-badge';
 import { CircleRegular } from '@fluentui/react-icons';
 import { mergeClasses, makeStyles, shorthands } from '@griffel/react';
 import { tokens } from '@fluentui/react-theme';
 
-const badgeColors: BadgeCommons['color'][] = [
+const badgeColors: BadgeProps['color'][] = [
```

Comment:
(nit) I think you could avoid the code changes in this file if you used `Required<BadgeProps>['color']` anywhere you need this type. Same for `Required<BadgeProps>['appearance']` elsewhere.

```suggestion
const badgeColors: Required<BadgeProps>['color'][] = [
```

And maybe define a type
```suggestion
type BadgeColor = Required<BadgeProps>['color'];
const badgeColors: BadgeColor[] = [
```